src/wallet/rpcwallet.cpp

```diff
@@ -1030,6 +1035,7 @@ UniValue getbalance(const UniValue& params, bool fHelp)
             "2. minconf          (numeric, optional, default=1) Only include transactions confirmed at least this many times.\n"
```

Comment:
The default should be 0. (In Bitcoin, "The default is 1 if an account is provided or 0 if no account is provided", and we do not allow providing an account in the Bitcoin sense.)